JapaneseFilter/sections/css_extended.txt

```diff
@@ -11,6 +11,7 @@ blog.livedoor.jp#?#.plugin-memo > div.side > div.yms-user-ad-side:upward(2)
 blog.livedoor.jp#?#font > #links_left > div.plugin-memo:has(a[href^="https://al.dmm.co.jp/"])
 blog.esuteru.com#?#blockquote > div.amazlet-box:upward(1)
 417ena.net#?#.archive__item > ins.adsbygoogle:upward(1)
+jpopblog.com##.sidebar_list > li#widget_text:has(> div.textwidget > center > a[href^="https://cryptoblog.biz/"][target="_blank"])
```

Comment:
better `center > a[target="_blank"] > img` instead of `> div.textwidget > center > a[href^="https://cryptoblog.biz/"][target="_blank"]`